task/02-numbers-tasks.js

```diff
@@ -202,7 +209,17 @@ function roundToPowerOfTen(num, pow) {
  *   17 => true
  */
 function isPrime(n) {
-    throw new Error('Not implemented');
+    if (n === 1) {
+        return false;
+    } else {
+        for (let x = 2; x < n; x++) {
+            if (n % x === 0) {
+                return false;
+            }
+        }
+
+        return true;
+    }
```

Comment:
We can avoid `else` statement and unnecessary nesting

```suggestion
    if (n === 1) {
        return false;
    }
    
    for (let x = 2; x < n; x++) {
        if (n % x === 0) {
            return false;
        }
    }

    return true;
}
```